### ISSUE TITLE
After updating ATM6 (which I added your mod to), everyones shops and wallets are now empty 

### ISSUE THREAD

**Issue by sprocketaudio** [Calemi/CalemiUtils-1.16.4#5]
Is there server side files I need to copy over to new updated pack? I have full backup of everything server side before update.

**Comment by sprocketaudio** [comment]
Adding a bit more to this, it seems one of the shops recognise who they are owned by, and all items have dissapeared.

**Comment by Calemi** [comment]
Sorry for the late reply. I'm not sure why this bug is happening. I'm saving the data to NBT, so there should be no files that need to be transferred over after updating.

**Comment by sprocketaudio** [comment]
nbt on the p[layer data? or somewhere else? I could do a comparison of before and after update onthe data if you let me know were to look

**Comment by sprocketaudio** [comment]
Any news on this? Or can u tell me where nbt data is? Is it in the region files? Block data? Player data?

**Comment by Calemi** [comment]
Sorry, I store the NBT data in the tile entity, so I guess that's within the block

**Comment by sprocketaudio** [comment]
just trying to think how this could happen.. im using a hosting service, and they do the modpack upgrade when i choose to upgrade.. I'm pretty sure after the upgrade finishes, it auto boots the server. If (hypothetically) it started, without your mod, i stopped it, added back your mod, then restarted.. could this be the cause? .. I do not know if this actually happened.. and i would have thought if it did boot once without your mod, all the market blocks would dissappear completely right? - which has not happened.

**Comment by sprocketaudio** [comment]
Hi Mate, I have got a bac to back comparison of before and afetr the server pack got upgraded - withouth changing your mod version..
![image](https://user-images.githubusercontent.com/13222732/107800756-195c6500-6d57-11eb-9217-dcd545685880.png)

**Comment by Calemi** [comment]
This is really strange. The ownerName tag is completely missing after the update. I have been playing with my mod on a modpack of my own and updated it many times, and our shops stay the same. (I host my own server, and I replace the mod and re-launch the server.)

It might be what you said about the hosting. I believe yes, the trading posts would disappear if the server was loaded without the mod installed. BUT I think the chunk would have to be loaded in order for it to disappear. So maybe it just removed some of the NBT data. But idk if that adds up...

**Comment by sprocketaudio** [comment]
i actually tested and semi resilved this in my head just today.. I loaded my server backup from before update as single player world in newer version of pack, with your mod in place.. and it was fine. I beleive this was the cause.. the update by the hosting company wipes the mods folder, and populates with new ones, then restarts automatically.. I did an update today and watched it with an FTP client, except I quickly popped your JAR in place before it finished putting all other update files in place and starting boot sequence.. I think this can be closed as its not really a bug! Thanks for your time though.
